cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,14 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  beforeEach(()=> cy.visit('https://conduit.mate.academy/user/login'));
 
-  });
+  it('should provide an ability to log in', () => {
+  cy.url().should('contain', 'login');
+  cy.get(':nth-child(1) > .form-control').type ('mate8568@ukr.net');
+  cy.get(':nth-child(2) > .form-control').type ('Q1w2e3r4');
```

Comment:
Same as above, avoid hardcoding values directly in the test cases. Store these values in a separate config file or environment variables.